src/config/color.c

```diff
@@ -59,7 +59,7 @@ static struct color_pair_cache
     struct
     {
         int16_t fg, bg;
-    } * pairs;
```

Comment:
For this please see https://github.com/profanity-im/profanity/pull/1774

Reply:
Done.